src/seccomp/src/lib.rs

```diff
@@ -880,6 +984,16 @@ impl SeccompFilter {
         Ok(Self {
             rules,
             default_action,
+            target_arch: target_arch.try_into().map_err(Error::Arch)?,
+        })
+    }
+
+    /// Creates an empty `SeccompFilter` which allows everything.
+    pub fn empty(target_arch: &str) -> Result<Self> {
```

Comment:
nit: Can this be a `Default` implementation? Do you think it makes sense from a semantic pov?